src/apps/account/services/user.service.ts

```diff
@@ -91,6 +108,7 @@ export class UserService {
       throw new AppException(errorCode, message, httpCode);
     }
 
+    data.password = await bcrypt.hash(data.password, 10);
```

Comment:
The password would be hashed from the auth service, don't need to hash password here. Pls remove this.